unicycle:  
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 200

  wheel_controller:
    type: effort_controllers/JointEffortController
    joint: wheel

  disc_controller:
    type: effort_controllers/JointEffortController
    joint: disc


